src/index.ts

```diff
@@ -163,7 +163,11 @@ function prepareLocalProvider ({ nuxt, options }, providerOptions) {
   providerOptions = defu(providerOptions, {
     baseURL: `http://${defaultHost}:${defaultPort}${prefix}`,
     internalBaseURL: `http://${defaultHost}:${defaultPort}${prefix}`,
-    dir: path.resolve(nuxt.options.srcDir, nuxt.options.dir.static)
+    dir: path.resolve(nuxt.options.srcDir, nuxt.options.dir.static),
+    placeholder: {
+      type: 'sqip',
```

Comment:
I think it would better if we use `image` as the default placeholder type and if make sqip a peer dependency.
What do you think?